### DISCUSSION TITLE
Can on_support allow FAQs to be transmitted?

### DISCUSSION THREAD

**Discussion by ravi-prakash-v** [beckn/protocol-specifications#277] [Core Specification]
Currently the `on_support` API returns a phone number, an email address or a chat link. Some platforms may provide only a FAQ document as support. Can we add that in the `support` schema?

@pramodkvarma @venkatramanm

**Comment by pramodkvarma** [comment]
In addition to phone/email/chat link, allow a general URL for sending back a general support URL, can be FAQ or simply a URL to support website or anything else for that matter.

**Comment by ravi-prakash-v** [comment]
But how will a BAP know how to call that URL? What format/structure the data will be returned? What method - HTTP/GET ot HTTP/POST etc?

**Comment by pramodkvarma** [comment]
Either HTTP URL or any standard acceptable URL like "mailto" "callto" or any deep link trigger. BAP can simply navigate it using an embedded browser. Some specific links like mailto etc an be rendered natively, but, beyond that any valid Internet/deeplink URL should work using embedded browser.
